### PR TITLE
DOC: fix minor typos

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: doc/release/1.14.0-notes.rst [hunk 1]
```diff
@@ -92,6 +92,9 @@ Future Changes
   This translation now gives a warning that explains what translation is
   occurring.  In the future, the translation will be disabled, and the first
   example will be made equivalent to the second.
+This translation now gives a warning explaining what translation is occurring.
```
Comment (charris) [inline doc/release/1.14.0-notes.rst]: Wrong indentation.

File: doc/release/1.14.0-notes.rst [hunk 2]
```diff
@@ -92,6 +92,9 @@ Future Changes
...
+In future, the translation will be disabled, and the first example will be made
```
Comment (charris) [inline doc/release/1.14.0-notes.rst]: Keeping "the" is better.

**Comment by charris** [comment]
Generally good. I don't think the release notes need fixing.

File: doc/release/1.14.0-notes.rst (hunk 1 above)
Comment (eric-wieser) [reply doc/release/1.14.0-notes.rst]: This entire block duplicates the lines above it...

Comment (luzpaz) [reply doc/release/1.14.0-notes.rst]: > This entire block duplicates the lines above it...

Not sure how this happened. Will remove this from the PR.

File: doc/release/1.14.0-notes.rst (hunk 2 above)
Comment (luzpaz) [reply doc/release/1.14.0-notes.rst]: Removing this commit from the PR as well since it is a duplicate line.

**Comment by luzpaz** [comment]
@charris do you prefer me to make another commit or should I squash the changes all in to 1 commit and force push ?

**Comment by charris** [comment]
Squash them together if it isn't a problem. We don't worry about changing history while the PR is in process.

**Comment by luzpaz** [comment]
@charris squashed and pushed

**Comment by charris** [comment]
The release notes are still messed up, just leave them alone. You can checkout the original file from master.

**Comment by eric-wieser** [comment]
I think this is #8603 biting again

**Comment by eric-wieser** [comment]
@charris: Release notes look untouched to me

**Comment by eric-wieser** [comment]
Thanks @luzpaz!

**Pull request merged**